src/app/iteration/iteration.component.html

```diff
@@ -11,17 +11,16 @@
         </h3>
       </div>
       <ul class="iteration-expanded-div" [collapse]="isCollapsedCurrentIteration">
-        <li class="active" *ngFor="let iteration of currentIterations" [routerLinkActive]="'selected-iteration'">
+        <li class="active" *ngFor="let iteration of currentIterations" [routerLinkActive]="'selected-iteration'" 
```

Comment:
So we had it almost like this previously, and that's the reason I changed them to anchor tag. It works fine, so if we think it is okay for the summit, then let's go ahead with this.